### ISSUE TITLE
Add credit for Citizen Engagement image for redesigned homepage

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3351]
### Overview
We need to add a credit for a new image used in the redesigned version of the homepage so that the [Credits page](https://www.hackforla.org/credits/) is up to date.

### Details
From the [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724), here is a screenshot of the credit's information for reference:
  <details>
    <summary>Click here to see the screenshot of the Citizen Engagement credit in Figma</summary>
    <img alt="" src="https://user-images.githubusercontent.com/31293603/178167459-3fd08c38-1a5b-4ba2-ad84-73a2991fcba9.png">
  </details>

### Action Items
- [x] **Feature Branch:** You will be working off a feature branch called `feature-homepage-launch`. Please follow ["Step 1: Create your branch" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-1-create-your-branch) wiki article.
- [x] In `_data/internal/credits` directory, create a file called `citizen-engagement2.yml`
- [x] In the file you just created, copy and paste the following:
```
---
# The title from the source provider
title: Citizen Engagement
# The url where you can get direct access to the image on the provider site
title-link: https://pixabay.com/photos/los-angeles-la-city-hall-6041398/
# The type of content - image, audio, or video 
content-type: image
# The page it's used on
used-in: Homepage
# Name of the artist
artist: Cnugent69
# Name of the image database you found it on
provider: Pixabay
# Please give the general site URL, not the link to the image
provider-link: https://pixabay.com
# Location in our GitHub repo
image-url: /assets/images/program-areas/citizen-engagement.png
# Alt text for the image on the Credits page
alt: ''
---
```
- [x] To update your topic branch, please follow ["Step 2: Update your branch with upstream changes" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-2-update-your-branch-with-upstream-changes) wiki article.
- [x] The pull request created from this issue should be merged into `feature-homepage-launch`. Please follow ["Step 3: Create a pull request" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-3-create-a-pull-request) wiki article
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3130 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please do the following in issue #3130:
    - [ ] Move it to the New Issue Approval column
    - [ ] Remove the "Dependency" label from it
    - [ ] Add a "ready for dev lead" label to it  

### Resources/Instructions
- This issue is tracked in the epic #3130.
- [Credits page](https://www.hackforla.org/credits/)
- [How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- [Credits yml file template for Iconography and Imagery](https://github.com/hackforla/website/wiki/Credits-yml-file-template-for-Iconography-and-Imagery)
- [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724)

**Comment by JessicaLucindaCheng** [comment]
This issue would usually have the milestone 01. Compliance. However, it's specifically for the homepage launch, which has not happened yet.

**Comment by lilyarj** [comment]
ETA: 8/30
Availability: 2hrs

**Comment by AzaniaBG** [comment]
ETA: 09/23/2022 by 5 PM
Availability: 09/23/2022 2:30 PM - 5 PM

**Comment by ExperimentsInHonesty** [comment]
Manually closing  (instead of the PR closing it on merge) this issue because its on a feature branch, so build in automations are not working.
